test/unit/test_formula.py

```diff
@@ -279,6 +287,56 @@ def test_generate_formula_no_test():
     assert 'test do' not in formula
 
 
+def test_generate_formula_multiline_test():
+    """Tests that we generate the formula content correctly (when there is no test).
```

Comment:
issue: We'll want this to properly reflect this test.

```suggestion
    """Tests that we generate the formula content correctly (when there are multi-line strings used).
```